will_of_the_prophets/board.py

```diff
@@ -79,11 +79,19 @@ def get_special_squares(now):
 class Square:
     """A square in the board."""
 
-    def __init__(self, number, now, row_reversed=False, is_current_position=False):
+    def __init__(
+        self,
+        number,
+        now,
+        row_reversed=False,
+        is_current_position=False,
+        was_visited=False,
```

Comment:
@craiga One of the code climate linters is complaining that there are now 5 arguments:

> Function __init__ has 5 arguments (exceeds 4 allowed). Consider refactoring. 

Again, my python chops aren't great. What would be the best way to refactor this?